README.md

```diff
@@ -17,13 +17,22 @@ background activity resumes normally.
 # Install
 
 1. Ensure your Kubernetes is using `containerd` as the container runtime.
-   - e.g. for minikube, add `--runtime containerd` to `minikube start` command.
+   - e.g. for minikube, add `--container-runtime containerd` to `minikube start` command.
+     ```bash
+     minikube start --memory=12000 --cpus=6 --kubernetes-version=v1.20.0 --vm-driver=kvm2 --disk-size=30g \
+     --extra-config=apiserver.enable-admission-plugins="LimitRanger,NamespaceExists,NamespaceLifecycle,ResourceQuota,ServiceAccount,DefaultStorageClass,MutatingAdmissionWebhook" \
+     --container-runtime=containerd
```

Comment:
Let's remove the bits from this that are system specific (e.g. I think kvm2 is just a linux thing and I _think_ the extra-config is some stuff in my copy/paste buffer that isnt actually needed)

```suggestion
     minikube start --kubernetes-version=v1.20.0 --container-runtime=containerd
```